sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,4 +1,45 @@
-﻿# Release History
+# Release History
+
+## 5.0.0-preview.2 (2019-08-06)
+
+### General
+
+- Public members of type `DateTime` were converted to `DateTimeOffset` to explicitly represent that they are UTC
+
+- Members of type `DateTimeOffset` have had the `Utc` suffix removed from their names, as the `DateTimeOffset` type removes ambiguity about the time zone
+
+- Members representing a partition offset that were incorrectly exposed as `string` have been converted to `int` to provide a unified type.
+
+- Renamed all instances of `EventHubPath` to `EventHubName` to align with the usage context and unify on the chosen semantics across the client library for different languages.
+
+- Ongoing minor tweaks to formatting, wording, and spelling of documentation and XML document comments.
+
+#### Publishing Events

```

Comment:
Consider switching to sentence-style capitalisation for headers. :D
https://docs.microsoft.com/en-ca/style-guide/capitalization

Reply:
see above.   I've done it, but I'm not at all in agreement with the style guide.